src/codecatalyst/model.ts

```diff
@@ -30,8 +29,12 @@ import { fileExists } from '../shared/filesystemUtilities'
 import { CodeCatalystAuthenticationProvider } from './auth'
 import { ToolkitError } from '../shared/errors'
 import { Result } from '../shared/utilities/result'
+import { VscodeRemoteConnection, ensureDependencies } from '../shared/remoteSession'
+import { VscodeRemoteSshConfig, sshLogFileLocation } from '../shared/vscodeRemoteSshConfig'
 
 export type DevEnvironmentId = Pick<DevEnvironment, 'id' | 'org' | 'project'>
+export const hostNamePrefix = 'aws-devenv-'
+export const connectScriptPrefix = 'codecatalyst_connect'
```

Comment:
No need to define this module-level. Should be a literal arg to `sshLogFileLocation`

Reply:
I copied this format from the existing CodeCatalyst code, but just refactored it to minimize the number of module-level constants. All module level constants were removed with the exception of `codecatalyst_connect` script since it is imported in the test files.